OpenRA.Game/Settings.cs

```diff
@@ -72,7 +72,7 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
-		public string TimestampFormat = "HH:mm";
+		public string TimestampFormat = "dd/MM - HH:mm:ss";
```

Comment:
Yes, I was afraid of suggesting too long string for prettiness-sake, "s" seems to be: YYYY-MM-ddTHH:mm:ss (No Z at the end, but could probably be added if needed)